File: matiasmasso/Mat.NET/DTO/DTOQtySku.cs
namespace DTO
{
    public class DTOQtySku
    {
        public int Qty { get; set; }
        public DTOProductSku Sku { get; set; } = null/* TODO Change to default(_) if this is not a reference type */;

        public DTOQtySku(int iQty, DTOProductSku oSku) : base()
        {
            Qty = iQty;
            Sku = oSku;
        }
    }
}

File: matiasmasso/Mat.NET/DT2/DTOQtySku.vb
Public Class DTOQtySku
    Property Qty As Integer
    Property Sku As DTOProductSku = Nothing

    Public Sub New(iQty As Integer, oSku As DTOProductSku)
        MyBase.New()
        _Qty = iQty
        _Sku = oSku
    End Sub

End Class
